### ISSUE TITLE
tslint errors with "missing trailing comma" in invalid location

### ISSUE THREAD

**Issue by shudson250** [palantir/tslint#4246]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: 3.1.3
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
// code snippet
hydrate(
  <BrowserRouter>
    <App />
  </BrowserRouter>,
  document.getElementById('root')
);
```

with `tslint.json` configuration:

```json
{
  "extends": ["tslint:recommended", "tslint-react"],
  "rules": {
    "arrow-parens": [true, "ban-single-arg-parens"],
    "indent": [true,"spaces"],
    "interface-name": [true, "never-prefix"],
    "jsx-no-lambda": false,
    "jsx-no-multiline-js": false,
    "member-access": false,
    "member-ordering": [false],
    "no-console": false,
    "no-duplicate-imports": true,
    "no-implicit-type": false,
    "no-inferrable-types": false,
    "no-unsafe-any": false,
    "no-var-requires": false,
    "object-literal-sort-keys": false,
    "only-arrow-functions": false,
    "ordered-imports": false,
    "prefer-const": true,
    "quotemark": [true,"single","jsx-double"],
    "semicolon": true,
    "variable-name": [true, "ban-keywords", "check-format", "allow-leading-underscore", "allow-pascal-case"]
  }
}
```

#### Actual behavior
tslint expects that a comma be inserted after the final argument of a function call.

I realise that this behavior can be modified in the config file, but why would function calls be included at all in this?

#### Expected behavior
I would expect tslint not to ask for a comma in a function call. Doing so leaves it open to interpretation that another (or follow-on) arguments are expected or available.

It looks like #2474 was similar to this issue, and was closed without resolution. Has anyone else run into this problem?

Thanks!

**Comment by shudson250** [comment]
Configurable...
